src/test/java/org/jitsi/meet/test/web/WebParticipantOptions.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.web;
+
+import org.jitsi.meet.test.base.*;
+
+import java.util.*;
+
+/**
+ * Holds web specific options.
+ */
+public class WebParticipantOptions
+    extends ParticipantOptions
+{
+    private static final String FAKE_AUDIO_PROP = "fakeStreamAudioFile";
+    private static final String FAKE_VIDEO_PROP = "fakeStreamVideoFile";
+    private static final String REMOTE_PROP = "isRemote";
+    private static final String BINARY_PROP = "binary";
+    private static final String VERSION_PROP = "version";
+
+    /**
+     * A prefix for global options (not per participant).
+     */
+    private static final String GLOBAL_PROP_PREFIX = "jitsi-meet";
+
+    /**
+     * Constructs new WebParticipantOptions, filling it with default values.
+     */
+    public WebParticipantOptions()
+    {
+        this.setProperty(FAKE_AUDIO_PROP, "resources/fakeAudioStream.wav");
```

Comment:
should there be also a default for FAKE_VIDEO_PROP (I don't remember if we've had one) ?

Reply:
Nope, we only have one for lipsync and psnr.